MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -1176,6 +1176,14 @@ public Date getDate() {
         return personnel;
     }
 
+    public ArrayList<Person> getActivePersonnel() {
```

Comment:
Public method needs javadoc comment.

Reply:
Whelp. Googling "how to javadoc", brb. 😛